examples/ransomware_detection/stages/create_features.py

```diff
@@ -59,10 +59,9 @@ def __init__(
         threads_per_worker: int = 2,
     ):
         self._client = Client(threads_per_worker=threads_per_worker, n_workers=n_workers)
+        self._feature_config = FeatureConfig(file_extns, interested_plugins)
+        self._feas_all_zeros = dict.fromkeys(feature_columns, 0)
```

Comment:
Is `self._feas_all_zeros` used anywhere else? Do we need to save it in `self`?

Reply:
using it here: https://github.com/nv-morpheus/Morpheus/blob/7b0bb90431921041954d4b5ebe75d23583110eeb/examples/ransomware_detection/stages/create_features.py#L114